www/server.js

```diff
@@ -123,8 +123,8 @@ server.route({
     ;
 
     var requestObject = {
-      to: 'hello@therefore.ca',
-      from: 'hello@therefore.ca',
+      to: 'homer@therefore.ca',
```

Comment:
@chandantherefore again, depending on the merges, please ensure this remains as `hello@`